data/forms/child-referral.jsx

```diff
@@ -0,0 +1,565 @@
+          label:
+            'Do you have any details of any other professionals or agencies working with the family: ?',
+          options: ['Yes', 'No'],
+        },
+        <h3 key="text" className="govuk-body">
+          Details of any other professionals or agencies working with the
+          family:
+        </h3>,
+        <p key="text" className="govuk-body">
+          (eg GP, health visitor, other professionals working with the
+          household)
+        </p>,
+        {
+          component: 'TextInput',
+          name: 'professional_name',
+          width: '30',
+          label: 'Name of professional (1)',
+          rules: { required: true },
+        },
+        {
+          component: 'TextInput',
+          name: 'professional_relationship',
+          width: '30',
+          label:
+            'Relationship of the professional to the child and family - professional (1)',
+        },
+
+        {
+          component: 'TextInput',
+          name: 'professional_telephone_1',
+          width: '30',
+          label: 'Telephone number - professional (1)',
+        },
+        {
+          component: 'TextInput',
+          name: 'professional_email_1',
+          width: '30',
+          label: 'Email address - professional (1)',
+        },
+        {
+          component: 'TextInput',
+          name: 'professional_address_1',
+          width: '30',
+          label: 'Address line 1 - professional (1)',
+        },
+        {
+          component: 'TextInput',
+          name: 'professional_address_2',
+          width: '30',
+          label: 'Address line 2 - professional (1)',
+        },
+        {
+          component: 'TextInput',
+          name: 'professional_address_3',
+          width: '30',
+          label: 'Address line 3 - professional (1)',
+        },
+        {
+          component: 'TextInput',
+          name: 'professional_postcode_1',
+          width: '30',
+          label: 'Postcode - professional (1)',
+        },
+        {
+          component: 'Radios',
+          name: 'professional_consent',
+          label:
+            'Professional (1): Has the person with parental responsibility consented for Hackney to make contact with this person or agency?',
+          options: ['Yes', 'No'],
+        },
+        {
+          component: 'Radios',
+          name: 'professional_other',
+          label:
+            'Do you have any more information about additional professionals working with the family',
+          options: ['Yes', 'No'],
+          rules: { required: true },
+        },
+      ],
+    },
+    {
+      id: 'requestForSupport',
+      title: 'Request For Support',
+      components: [
+        <h3 key="text" className="govuk-body">
+          Hackney Child Wellbeing Framework
+        </h3>,
+        <p key="text" className="govuk-body">
+          Please consult the Hackney Child Wellbeing Framework
+          https://tinyurl.com/yxzq446e for guidance around levels of
```

Comment:
Should this url be a link or it's fine as text?

Reply:
I will ask. It's like this in the google form.

Reply:
used Link component, currently opens in the same window, will check with UX if it needs to be a new tab

Reply:
`Link` component is only for internal links (because it's working in conjunction with the Router).
If this is meant to be a link I would have it open in a new tab.